docs/pages/accordion.md

```diff
@@ -125,6 +125,39 @@ By default, at least one pane in an accordion must be open. This can be changed
 
 ---
 
+### Disabled
+
+There may be times where you want to disable pane switching on an accordion. This can be accomplished by setting the disabled attribute.
+
+```html
```

Comment:
I used the rest of the file as the basis for the modification.  No where in the rest of the file is `html_example` used.  If this needs to be changed, I can do that.